Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonOverviewFragment.kt

```diff
@@ -79,15 +80,18 @@ class PersonOverviewFragment : Fragment(R.layout.person_overview_fragment), Auto
                 )
             }
 
-            is ShowDeleteDialog -> MaterialAlertDialogBuilder(requireContext())
-                .setTitle(R.string.test_certificate_delete_dialog_title)
-                .setMessage(R.string.test_certificate_delete_dialog_body)
-                .setNegativeButton(R.string.test_certificate_delete_dialog_cancel_button) { _, _ -> }
-                .setCancelable(false)
-                .setPositiveButton(R.string.test_certificate_delete_dialog_confirm_button) { _, _ ->
-                    viewModel.deleteTestCertificate(event.containerId)
-                }
-                .show()
+            is ShowDeleteDialog -> DialogHelper.showDialog(
```

Comment:
I think you need to use the new builder `displayDialog` here ?
@SamuraiKek , right?

Reply:
Changed to new builder